tools/releases/dockerfiles/kumactl.Dockerfile

```diff
@@ -5,3 +5,5 @@ ARG ARCH
 # override NOTICE
 COPY /tools/releases/templates/NOTICE-kumactl /kuma/NOTICE
 COPY /build/artifacts-linux-$ARCH/kumactl/kumactl /usr/bin/
+
+ENTRYPOINT ["/busybox/busybox", "sh", "-c"]
```

Comment:
Why not `ENTRYPOINT ["/usr/bin/kumactl"]`?

Reply:
I wanted to be consistent with the older images

Reply:
Good point. I think the `kumactl` entrypoint is probably the "expected" one for an image like this, though? Maybe worth just changing and adding an UPGRADE note. Either way is fine with me.

Reply:
This should be backported, so let's make 2.1.x and 2.2.x consistent with the older images.
I'll do followup PR with suggested changes for 2.3.0, as after quick thinking I agree with it